specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/examples/createOrUpdateGuestDiagnosticSettings.json

```diff
@@ -0,0 +1,241 @@
+{
+    "parameters": {
+        "subscriptionId": "187f412d-1758-44d9-b052-169e2564721d",
+        "resourceGroupName": "Default-ResourceGroup",
+        "diagnosticSettingsName": "SampleDiagSetting",
```

Comment:
Suggestion: instead of SampleDiagSetting, use "productionMachineSetting" , which is more like a real customer scenario (one setting for all production machines)